src/core/core.tooltip.js

```diff
@@ -492,8 +492,25 @@ function getBeforeAfterBodyLines(callback) {
 
 var exports = Element.extend({
 	initialize: function() {
-		this._model = getBaseModel(this._options);
-		this._lastActive = [];
+		var me = this;
+		me._model = getBaseModel(me._options);
+		me._view = {};
+		me._lastActive = [];
+	},
+
+	transition: function(easingValue) {
+		var me = this;
+		var options = me._options;
+
+		if (me._lastEvent && me._chart.animating) {
```

Comment:
I get why this is happening, but I'm not sure if inside of `transition` is the best way to achieve it. @kurkle @nagix any thoughts?

Reply:
Any progress on this?

Reply:
I can't think of a better place (that exists).